Simplified/NYPLCatalogNavigationController.m

```diff
@@ -102,8 +102,8 @@ - (void)switchLibrary
 #if defined(FEATURE_DRM_CONNECTOR)
       if([NYPLADEPT sharedInstance].workflowsInProgress) {
         [self presentViewController:[NYPLAlertController
-                                     alertWithTitle:@"SettingsAccountViewControllerCannotLogOutTitle"
-                                     message:@"SettingsAccountViewControllerCannotLogOutMessage"]
+                                     alertWithTitle:@"PleaseWait"
+                                     message:@"PleaseWaitMessage"]
```

Comment:
Yes this change was already made in My Books and My Reservations, and I just missed it for the Catalog VC. It may not yet be translated to non-english though.